README.md

```diff
@@ -1,61 +1,436 @@
 # Argo Toolbox API
 
-## Docker compose
+A step‑by‑step guide to install, run, and explore the **Toolbox API** locally.  
+This documentation is tailored for **non‑technical users**, with simple instructions and no prior Docker experience required.
 
-This Docker Compose configuration allows you to run the Argo Toolbox API locally on your laptop, replacing the Kubernetes deployment for easier local development and testing.
+---
 
-### Prerequisites
+## 🚀 Quick start
 
-- Docker (Docker Desktop for Windows)
+If you already have Docker and Git configured on your local machine, you can just follow these steps:
 
-### Quick start
+- clone this git repository :
 
-Clone this repository, rename the `.env.demo` file in `.env` and do this command :
+  ```bash
+  git clone https://github.com/amrit-eu/Argo-Toolbox.git
+  cd Argo-Toolbox
+  ```
+
+- rename file `.env.demo` in `.env`.
+
+- run docker compose to launches services :
+
+  ```bash
+  docker compose up -d
+  ```
+
+## 🖥️ Detailed local Deployment Guide
+
+This section provides installation instructions for Windows, Mac, and Linux. Follow the steps relevant to your operating system.
+
+### 🛠 Prerequisites
+
+Regardless of the operating system, you will need **Docker** and **Git**.
+
+
+#### 1. Install Docker
+
+- **Windows and Mac:**
+
+  - Download Docker Desktop: [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+  - Install with default settings and restart if prompted.
+  - Ensure Docker shows **Running**.
+
+- **Linux:**
+
+  - Update package index:
+
+    ```bash
+    sudo apt update
+    ```
+
+  - Install Docker:
+
+    ```bash
+    sudo apt install docker.io
+    ```
+
+  - Enable Docker:
+
+    ```bash
+    sudo systemctl enable docker
+    sudo systemctl start docker
+    ```
+
+  - Verify installation:
+
+    ```bash
+    docker --version
+    ```
```

Comment:
I'm tempted to say we should skip this for now – this is specific to Debian-based distros and doesn't deal with scenarios where the Docker repos need adding, etc., which I think is probably too much for a non-technical guide.